contrib/bsp/src/mill/contrib/bsp/ModuleUtils.scala

```diff
@@ -124,7 +124,6 @@ object ModuleUtils {
     }
     buildTarget.setData(dataBuildTarget)
     buildTarget.setDisplayName(module.millModuleSegments.render)
-    buildTarget.setBaseDirectory(module.intellijModulePath.toNIO.toUri.toString)
```

Comment:
Synced with @jastice and this is not needed and cause issues with cross compiled modules in Intellij.